.setup/bin/setup_sample_courses.py

```diff
@@ -1069,6 +1069,10 @@ def __init__(self, gradeable):
             self.team_assignment = False
             self.max_team_size = 1
             self.team_lock_date = submitty_utils.parse_datetime(gradeable['eg_submission_due_date'])
+            self.student_view = True
+            self.student_submit = True
+            self.student_download = False
+            self.student_any_version = True
```

Comment:
Sorry, I meant it doesn't look possible to see these values within the course.yml file.